testing/e2e/modules/types_registry/test_registration_debug_logging.py

```
@@ -0,0 +1,439 @@
+        # Instance should fail validation due to wrong type for 'age'
+        if new_logs and results[1]["status"] == "error":
+            assert "error" in results[1]
+            # Check if debug logging is enabled
+            debug_enabled = "DEBUG" in new_logs and "types_registry" in new_logs
+            if debug_enabled:
+                # Verify instance content is logged
+                assert "not_a_number" in new_logs, \
+                    "Expected instance content 'not_a_number' in debug logs"
+                # Verify GTS ID is logged
+                assert instance_id in new_logs or type_id in new_logs, \
+                    f"Expected GTS ID in debug logs"
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)